examples/delay-pod.js

```diff
@@ -0,0 +1,81 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { PodDisruptor } from '../src/pod.js';
+import { DeploymentHelper } from '../src/helpers.js';
+import  http from 'k6/http';
+
+const app = "httpbin"
+const image = "kennethreitz/httpbin"
+
+export function setup() {
+  const k8sClient = new Kubernetes()
+
+  // create a random namespace
+  const namespace = "k6-"+Math.random().toString(32).slice(2, 7);
+  k8sClient.namespaces.create({name: namespace})
+
+  // create a test deployment
+  const helper = new DeploymentHelper(k8sClient, app, namespace, image, 2)
+  helper.deploy()
+  
+  // give time for deployment's pods to be created
+  sleep(5)
```

Comment:
Should we require this? Or, by default, block, until Deployment is, e.g., ready.

Reply:
Yes, that would be deferentially better, as I did in the `expose` method that waits for the service to get its IP.

Reply:
Awesome 👍